### DISCUSSION TITLE
Separate Windows

### DISCUSSION THREAD

**Discussion by ontap-lab** [orbitalquark/textadept#31] [General]
Hi , I have just started using Textadept , on the new mac-mini M1 with Big Sur.
I cant figure out how to open two seperate Textadept windows at the same time so that I can edit and compare xml files side by side?
Thank you.
Joe R.

**Comment by orbitalquark** [comment]
Textadept uses a single window by default. You have to pass a command line argument, -f or --force, to the application to open a new window. I know on Windows and Linux you can create a shortcut to run Textadept with specific command line arguments, so perhaps the same functionality exists on macOS.

Alternatively you can use split views to view two buffers side-by-side. If you use the file diff module[1], you can visualize the differences between two buffers graphically.

[1]: https://github.com/orbitalquark/textadept-file-diff
